core-service/README.md

```diff
@@ -12,6 +12,6 @@ Menu is a list of dishes with prices. The `Menu is divided into categories (main
 
```

Comment:
Let's keep only short description (1-2 sentences ) about the service + link to the product requirements